whisper.cpp

```diff
@@ -2890,24 +2889,23 @@ int whisper_tokenize(struct whisper_context * ctx, const char * text, whisper_to
 }
 
 int whisper_lang_max_id() {
-    auto max_id = 0;
-    for (const auto & kv : g_lang) {
-        max_id = std::max(max_id, kv.second.first);
-    }
-
-    return max_id;
+    using it = decltype(*g_lang.cbegin());
+    return std::max_element(
+               g_lang.cbegin(), g_lang.cend(),
+               [](it x, it y) { return x.second.first > y.second.first; })
+        ->second.first;
```

Comment:
All changes starting from here to the end are not necessary - it just makes harder to read the code